### ISSUE TITLE
Any plan to adapt diesel 2? --> Yes, feature/diesel-2 branch is available.

### ISSUE THREAD

**Issue by NateLing** [bikeshedder/deadpool#194]
Diesel 2.0.0-rc.0 is out, I plan to adapt my project to v2, but deadpool not support diesel v2.

**Comment by bikeshedder** [comment]
Please give the `feature/diesel-2` branch a try: https://github.com/bikeshedder/deadpool/tree/feature/diesel-2

If needed I can release a RC on crates.io as well. If there is no hurry you can just add this branch to your `Cargo.toml`:

```toml
[dependencies.deadpool-diesel]
git = "https://github.com/bikeshedder/deadpool"
branch = "feature/diesel-2"
features = ["sqlite"]
```

**Comment by bikeshedder** [comment]
I totally missed the release of Diesel 2.0.0. I'm preparing a stable release of `deadpool-diesel 0.4.0` right now.

**Comment by bikeshedder** [comment]
I just released [`deadpool-diesel 0.4.0`](https://crates.io/crates/deadpool-diesel/0.4.0) on crates.io.
